config/config.example.yml

```diff
@@ -1,11 +1,147 @@
+# Invidious configuration file
+---
+# Number of threads to use for crawling videos from channels
 channel_threads: 1
+# Number of threads to use for refreshing subscription feeds
 feed_threads: 1
+
+# Log file path or STDOUT
+output: STDOUT
+
+# Log verbosity
+# All   = 0, Trace = 1
+# Debug = 2, Info  = 3
+# Warn  = 4, Error = 5
+# Fatal = 6, Off   = 7
+log_level: 3
+
+# Database configuration
 db:
   user: kemal
   password: kemal
   host: localhost
   port: 5432
   dbname: invidious
+
+# Use polling to keep decryption function up to date
+decrypt_polling: true
+
+# Crawling channel videos, threads should refresh
+# all videos uploaded by a channel
 full_refresh: false
+
+# Define if accessed via HTTPS
 https_only: false
-domain:
+
+# Signing key for CSRF tokens
+# randomly generated by default
+#hmac_key: 
+
+# External domain
+domain: replace.me
+
+# Use server-side notifications provided by YouTube
+# https://developers.google.com/youtube/v3/guides/push_notifications
+# Requires domain and hmac_key to be set
+use_pubsub_feeds: false
+
+# Show popular videos on this instance on the front page
+popular_enabled: true
+
+# Determine if CAPTCHA should be required for registration
+captcha_enabled: true
+
+# User login
+login_enabled: true
+
+# User registration
+registration_enabled: true
+
+# Instance statistics
+# Available on /api/v1/stats
+statistics_enabled: false
```

Comment:
More information should be added to explain what this shows, along with mentioning that it's useful to have for instances.invidio.us (and should be therefore enabled for public instances).